src/Nutgram.php

```diff
@@ -201,6 +201,16 @@ public function run(): void
         $this->container->get(RunningMode::class)->processUpdates($this);
     }
 
+    /**
+     * @throws ContainerExceptionInterface
+     * @throws NotFoundExceptionInterface
+     */
+    public function once(): void
+    {
+        $this->preflight();
+        $this->container->get(RunningMode::class)->processUpdate($this);
+    }
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏This is doing the same thing as the normal run method, so I'm not sure whats going on here

Reply:
this processes only one update at each time called.

Reply:
Yeah, but thats exactly the same code of the `run()` method
https://github.com/nutgram/nutgram/pull/712/files#diff-ca41639f8e11a0ef59e5efa79cd1d1b469893121cc6f2e961860004395dcad10L198-L202

Reply:
the difference is so small and hard to notice. I added `processUpdate` (singular name) to the pulling mode in addition to `processUpdates` (plural) function. so we can only process one update.